resotolib/resotolib/baseplugin.py

```diff
@@ -133,10 +134,10 @@ def action_processor(self, message: Dict) -> None:
 
     def run(self) -> None:
         try:
-            ArgumentParser.args = self._args
-            resotolib.config._config = self._config
-            setup_logger("resotoworker")
-            resotolib.signal.initializer()
+            # ArgumentParser.args = self._args
+            # resotolib.config._config = self._config
+            # setup_logger("resotoworker")
+            # resotolib.signal.initializer()
```

Comment:
can be removed?

Reply:
I want to eventually go back to using Process so I'm keeping it here for reference of what's needed then.